File: delivery/DeliveryCompleted.py
from AbstractEvent import AbstractEvent
import json
from datetime import datetime

class DeliveryCompleted(AbstractEvent):
    id : int
    orderId : str
    status : str
    
    def __init__(self):
        super().__init__()
        self.id = None
        self.orderId = None
        self.status = None

File: customercenter/src/main/java/shopmyid/domain/DeliveryCompleted.java
package shopmyid.domain;

import java.time.LocalDate;
import java.util.*;
import lombok.Data;
import shopmyid.infra.AbstractEvent;

@Data
public class DeliveryCompleted extends AbstractEvent {

    private Long id;
    private String orderId;
    private String status;
}
